client/state/themes/selectors.js

```diff
@@ -60,6 +60,22 @@ export const getTheme = createSelector(
 );
 
 /**
+ * Returns theme request error object
+ *
+ * @param  {Object}  state   Global state tree
+ * @param  {String}  themeId Theme ID
+ * @param  {Number}  siteId  Site ID
+ * @return {Object}          error object if present or null otherwise
+ */
+export function getThemeRequestsError( state, themeId, siteId ) {
+	const themes = get( state.themes.themeRequestsError, siteId, null );
```

Comment:
The variable name is misleading here since we're getting an error, not a `themes` array.

I think the whole selector could be shortened to

```js
return get( state.themes.themeRequestsError, [ siteId, themeId ], null );
```

thanks to the way `get` works. (You could consider also dropping the fallback `null` -- it will then just return `undefined`, which should be expressive enough too, I think.)